### ISSUE TITLE
Alt text for images in blog posts

### ISSUE THREAD

**Issue by mine-cetinkaya-rundel** [rstudio/education.rstudio.com#146]
We should have alt text for images in blog posts and set `fig_caption: false` (or whatever the equivalent for that is for blogdown). I perused a couple of blog posts and didn't see them in use.

**Comment by apreshill** [comment]
Agreed! Happy to merge in PRs to add them to past posts.

**Comment by mine-cetinkaya-rundel** [comment]
Ok I'll do this in the next couple weeks! I'm thinking a single PR to add alt text to all post posts.

**Comment by mine-cetinkaya-rundel** [comment]
For figures created in R chunks, we can use the method described at https://ropensci.org/technotes/2020/04/23/rmd-learnings/. For embedded images just html alt text should do. 

For the teaching tidyverse posts I added the knitr hook to the setup chunk of the Rmd. 

@apreshill Is there a way to centrally include that, or is including the knitr hook definition in the setup chunk for each post the way to go? 
 
Once we have a workflow I can also add instructions to the README.
